front/app/translations/admin/en.json

```
@@ -1452,6 +1453,14 @@
   "app.containers.admin.ideas.all.inputManagerPageTitle": "Input manager",
   "app.containers.admin.ideas.all.tabManage": "Edit",
   "app.containers.admin.ideas.all.tabStatuses": "Statuses",
+  "app.containers.admin.import.downloadTemplate": "Download template",
+  "app.containers.admin.import.importDescription": "Import existing inputs from an Excel file.",
+  "app.containers.admin.import.importHint": "Important: In order to send the invitations correctly, no column can be removed from the import template. Leave unused columns empty.",
```

This file is generated via CrowdIn, so this is not a relevant comment. I just want to indicate that the changes elsewhere should be reflected in this file (and front/app/translations/en.json below).

```suggestion
  "app.containers.admin.import.importHint": "Important: In order to send the ideas correctly, no column can be removed from the import template. Leave unused columns empty.",
```